aws/route53/test_route53_zones.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+
+from aws.route53.resources import zones, cnames
+from helpers import get_param_id
+
+
+@pytest.mark.parametrize(
+    "cnames", cnames(), ids=lambda record: get_param_id(record, "Name")
+)
+def test_route53_cnames(cnames):
+    assert int(cnames["TTL"]) >= 600, f"TTL is {cnames['TTL']}"
```

Comment:
I'd like to see a docstring for this test explaining why we are enforcing `600` here. As well, in the assert message, it might be nice to have something like `TTL is expected to be 600 or greater, but it is X` or something similar.